packages/feedback/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,9 @@
+export const environment = {
+  production: true,
+  name: '',
+  api: '',
+  graphqlAPI: '',
+  client: '',
+  subscription: '',
+  authorization: '',
```

Comment:
The `gitlabURL` field is not present in this file, but is in the qa, stage and normal environment files.